jekyll/faq.md

```diff
@@ -37,12 +33,20 @@ in detail and can typically be managed with minimal effort. The compiler also
 highlights deprecated features to provide sufficient notice and transition time
 through changes.
 
-## How fast is Nim?
 
-Benchmarks show it to be comparable to C or C++. Nim can be used as a high level
-language as well as a low level language -- always with excellent performance,
-but some language features are more expensive than others.
+## How about security and memory safety?
+
+Nim provides memory safety by not performing pointer arithmetic, with
+optional [checks](https://nim-lang.org/docs/manual.html#pragmas-compilation-option-pragmas),
+traced and untraced references, safe parallelism through a
+[disjoint checker](https://nim-lang.org/docs/manual.html#parallel-spawn-parallel-statement)
```

Comment:
The disjoint checker is very cool but not production ready to be mentioned here. And its future lies in a linting tool, independent from the compiler.